admin/class-yoast-form.php

```diff
@@ -153,7 +153,28 @@ public function set_options_value( $key, $value, $overwrite = false ) {
 	 */
 	public function admin_footer( $submit = true, $show_sidebar = true ) {
 		if ( $submit ) {
+			$floating_button = new WPSEO_Admin_Saved_Changes_Listener();
```

Comment:
I think `WPSEO_Admin_Settings_Saved` or `WPSEO_Admin_Settings_Saved_Listener` might be a better name for this.